### ISSUE TITLE
Cannot Access `Reset` Button in Interactive Window Only Using Keyboard

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/nodejstools#1345]
In the Node.js interactive window, there does not seem to be any way to get to the `reset` button without using a mouse. The `Clear` option appears in the context menu and can be accessed with `shift + f10`, but the `reset` option does not appear there.

**Comment by mjbvz** [comment]
You can type `.reset` to accomplish this, but this behavior is not very discoverable

**Comment by paulvanbrenk** [comment]
Use [Shift] + [Alt]
